From 57b77959b2eec1d7c5dfdfc360fbe4a79e072b1b Mon Sep 17 00:00:00 2001
From: caiyang <caiyang5@lenovo.com>
Date: Fri, 10 Oct 2025 15:34:38 +0800
Subject: [PATCH] [HQRSE-205]:[fix]fix gpio in pinctrl.dtsi[1/1]

[scope of influence]: lcd

[fix]:fix gpio in pinctrl.dtsi

[RootCause]:modify the default configuration of gpio in sleep mode

[drv]yes

[fac]yes

[dev]yes

[Solution]:Configuration gpio in pinctrl.dtsi

Change-Id: Ib8d0c056f8cf79346071eb57b7d736034b2f2e59
Reviewed-on: https://vcsc.lenovo.com:8080/c/device/lenovo/vendor_overlay/+/16893
Tested-by: Yang Cai <caiyang5@lenovo.com>
Reviewed-by: LiCheng Wang <wanglc13@lenovo.com>
Reviewed-by: Keshuang Lou <louks1@lenovo.com>
---

diff --git a/lapis/boot_images/boot/Settings/Soc/Lanai/Core/SocInfra/TLMM/pinctrl.dtsi b/lapis/boot_images/boot/Settings/Soc/Lanai/Core/SocInfra/TLMM/pinctrl.dtsi
index 8166040..a3e3061 100755
--- a/lapis/boot_images/boot/Settings/Soc/Lanai/Core/SocInfra/TLMM/pinctrl.dtsi
+++ b/lapis/boot_images/boot/Settings/Soc/Lanai/Core/SocInfra/TLMM/pinctrl.dtsi
@@ -1330,7 +1330,7 @@
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
-(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
+(GPIO_OUTPUT | GPIO_NO_PULL | GPIO_OUT_LOW | GPIO_PRG_YES) //GPIO3 GPIO_NO_PULL
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
@@ -1406,7 +1406,7 @@
 
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
-(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
+(GPIO_OUTPUT | GPIO_NO_PULL | GPIO_OUT_LOW | GPIO_PRG_YES)//GPIO72 GPIO_NO_PULL
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
@@ -1421,7 +1421,7 @@
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_YES) //GPIO84 GPIO_NO_PULL
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
-(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
+(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_YES) //GPIO86 GPIO_PULL_DOWN
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
@@ -1473,7 +1473,7 @@
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
-(GPIO_INPUT | GPIO_NO_PULL | GPIO_OUT_LOW | GPIO_PRG_NO)
+(GPIO_INPUT | GPIO_NO_PULL | GPIO_OUT_LOW | GPIO_PRG_YES)//GPIO133 GPIO_NO_PULL
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
@@ -1525,8 +1525,8 @@
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 
-(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
-(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
+(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_YES) //GPIO_180 GPIO_PULL_DOWN
+(GPIO_OUTPUT | GPIO_NO_PULL | GPIO_OUT_LOW| GPIO_PRG_YES) //GPIO181 GPIO_NO_PULL
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
@@ -1545,7 +1545,7 @@
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
-(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
+(GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_YES) //GPIO199 GPIO_PULL_DOWN
 
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
 (GPIO_INPUT | GPIO_PULL_DOWN | GPIO_OUT_LOW | GPIO_PRG_NO)
